src/BUILD

```diff
@@ -537,7 +537,8 @@ sh_binary(
         "tmp/build",
         "--platform",
     ] + select({
-        "//src/conditions:darwin": ["darwin"],
+        "//src/conditions:darwin_x86_64": ["darwin_x86_64"],
```

Comment:
Yes, I was just experimenting. This is looking a little more involved because of bzlmod. Will probably need a careful dance of interleaved releases / updates to rules_java, java_tools and bazel.